keyboards/cutie_club/novus/readme.md

```diff
@@ -0,0 +1,19 @@
+# novus
+
+![novus](https://i.imgur.com/kuCDFb0.png)
```

Comment:
```suggestion
![novus](https://i.imgur.com/kuCDFb0h.png)
```

Reply:
@drashna is this to ensure images dont end up too big in readme files?

 happy to apply the changes just curious :smiley_cat:

Reply:
Yes.